weather/ngsild-payloads/weatherAlert.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+	"id": "urn:ngsi-ld:WeatherAlert:Menton",
```

Comment:
since the finer granularity is the department, it should be `urn:ngsi-ld:WeatherAlert:AlpesMaritimes` (or `urn:ngsi-ld:WeatherAlert:Department:06` or ...)